knowledge/responses/cant_we_just_improve_the_welfare_of_animals.md

```diff
@@ -1 +1,14 @@
\ No newline at end of file
+Most consumers of animal products claim to care about the well-being of animals and will even agree that the way we raise and kill animals is cruel. This is partly why many people choose to boycott  pay extra for free-range eggs and animal products deemed organic or high welfare[1,2].
+
+However, the terms “free-range” or “humane” and other marketing terminology is an attempt to convince customers that their product is more ethical and morally righteous. In reality, welfare is highly subjective, incremental, and difficult to implement and enforce in reality. For instance, the US Department of Agriculture defines free-range as “free and continuous” access to the outdoors for at least 51% of a bird’s life. This means that thousands of birds have access to a few exits to small concrete porches without vegetation[3]. Even for European Union countries that are viewed to have high benchmarks for animal welfare laws, many of these countries fall short of adhering to them[4]. Welfarism only serves to ease the conscience of the consumer and reinforces the idea that there is an ethical way to exploit an animal.
+
+Regardless of the privilege or treatment received by the animals, there is no ethical or humane way to treat animals destined to be exploited and slaughtered. Improving the treatment of animals is better than maintaining the status quo, but it does not provide any substantial relief to the animals themselves and does not address or eradicate the problem of animal abuse and exploitation. Animals don’t want bigger cages or larger barns. All they want is to be free, not “free-range,” but free to live their lives in their entirety, without fear of human-inflicted pain, suffering or exploitation.
+
+### References
+[1] [Alonso ME, González-Montaña JR, Lomillos JM. Consumers' Concerns and Perceptions of Farm Animal Welfare. Animals (Basel). 2020 Feb 27;10(3):385. doi: 10.3390/ani10030385. PMID: 32120935; PMCID: PMC7143148.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7143148/#B11-animals-10-00385)
```

Comment:
Is this the correct citation format?

Reply:
Sure, I don't think it's too important to be perfect with references, as long as they're there and have a link